src/instructlab/log.py

```diff
@@ -68,7 +68,7 @@ def configure_logging(
     stream = logging.StreamHandler()
     stream.setFormatter(CustomFormatter(fmt=fmt))
     root.addHandler(stream)
-    root.setLevel(log_level)
+    logging.getLogger("instructlab").setLevel(log_level)
```

Comment:
```suggestion
    logging.getLogger(__name__.split(".", 1)[0]).setLevel(log_level)
```

I think something like this would be preferable to hard coding "instructLab".